### ISSUE TITLE
[utils/mysql] Replace Global Master Password

### ISSUE THREAD

**Issue by BugRoger** [sapcc/helm-charts#2294]
https://github.com/sapcc/helm-charts/blob/master/openstack/utils/templates/_hosts.tpl#L130

- Remove all references to `Values.global.master_password` 
- Prefer to not reference a global value. Instead require a password being passed to the chart

Prior Art:
https://github.com/sapcc/helm-charts/commit/a4c48f4160d81b1f3bbdb356f7526c431763ca4a
Take note that this utility function is being used in many other charts:
- `utils.password_for_fixed_user_mysql`
  - `utils.password_for_user_mysql`
    - `utils.root_password`
      - `db_url_mysql`
      		- https://github.com/sapcc/helm-charts/blob/master/openstack/nova/templates/bin/_db-update-cells.tpl
      		- https://github.com/sapcc/helm-charts/blob/master/openstack/barbican/templates/etc/_barbican.conf.tpl
      		- https://github.com/sapcc/helm-charts/blob/master/openstack/octavia/templates/etc/_octavia.conf.tpl
      		- https://github.com/sapcc/helm-charts/blob/master/openstack/neutron/templates/etc/_neutron.conf.tpl
      		- https://github.com/sapcc/helm-charts/blob/master/openstack/designate/templates/etc/_designate.conf.tpl

**Comment by BugRoger** [comment]
@Carthaca @rajivmucheli We still have a transitive dependency to the `master_password` in this utility function.

I suggest we delete the function:
https://github.com/sapcc/helm-charts/blob/0c26619ebb462261149801e561b0ec8342306fde/openstack/utils/templates/_hosts.tpl#L125-L131

and replace it with a similar `require` as was used here:

```
required ".Values.global.bigip_user required!") ( $envAll.Values.global.bigip_password | required ".Values.global.bigip_password required!"
```

**Comment by rajivmucheli** [comment]
Hi @BugRoger,

1. cinder & nova service need to merge/rollout the PR's mentioned in #2293, until then we cannot fix #2294.
2. The above format is already being implemented in #2293, example : https://github.com/sapcc/helm-charts/pull/2348/files
